bundles/org.eclipse.mita.base/src/org/eclipse/mita/base/typesystem/infra/MitaTypeLinker.xtend

```diff
@@ -44,7 +44,7 @@ class MitaTypeLinker extends Linker {
 	}
 	
 	override protected isNullValidResult(EObject obj, EReference eRef, INode node) {
-		return true;
+		return false;
```

Comment:
This makes the linker create issues if types are not found. This is necessary so that size inference does not happen if types are not properly linked, since a precondition of size inference is an error free typing, for simplicity.